payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/autocomplete/model/TransformGoogleToStripeAddress.kt

```diff
@@ -0,0 +1,259 @@
+package com.stripe.android.ui.core.elements.autocomplete.model
+
+import android.content.Context
+import android.os.Build
+import java.util.Locale
+
+// Largely duplicated from
+// https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/1c111621/src/elements/inner/autocomplete_suggestions/utils/transformGoogleToStripeAddress.ts#L248-L325
+
+internal data class AddressLine1(
+    var streetNumber: String? = null,
+    var route: String? = null,
+    var subLocalityLevel2: String? = null,
+    var subLocalityLevel3: String? = null,
+    var subLocalityLevel4: String? = null
+)
+
+internal data class Address(
```

Comment:
I wonder if we can share the `ShippingAddress` class I made here?

Reply:
This class is pretty much internal to this file, I don't think we need to expose this.